redmine/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+
+services:
+
+  redmine:
+    image: redmine
+    ports:
+      - 8080:80
+    environment:
+      REDMINE_DB_MYSQL: mysql
```

Comment:
Do you think it'd be smart to change this name to `db` or something so that it's more clear that this is a hostname?  I probably have updated that script myself and was still scratching my head at what this was for a bit. :innocent: